environment

```diff
@@ -37,6 +37,7 @@ OPENID_PROVIDER=humancellatlas.auth0.com
 ACM_CERTIFICATE_IDENTIFIER="826dbdb8-2b23-4cc9-8fd6-73aa6fc658d7"
 FUS_TERRAFORM_BACKEND_BUCKET_TEMPLATE=terraform-tsmith
 FUS_ADMIN_EMAILS=test_email@domain.com
+FUS_AUDIENCE=https://dev.data.humancellatlas.org/
```

Comment:
do the other env. files also have to be updated?

Reply:
only prod